src/middleware/validation.js

```diff
@@ -0,0 +1,14 @@
+export const validateRequest = (schema, property) => {
+  return (req, res, next) => {
+    const { error } = schema.validate(req[property]);
```

Comment:
Chamando dessa forma, vai retornar só o primeiro erro de validação. Pra retornar todos:
```suggestion
    const { error } = schema.validate(req[property], { abortEarly: false });
```
https://joi.dev/api/?v=17.3.0#anyvalidatevalue-options